<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QDialog" name="main_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Умные заметки</string>
  </property>
  <widget class="QTextEdit" name="text_of_note">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>24</y>
     <width>671</width>
     <height>671</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="notes_label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>25</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Список заметок </string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_note_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>250</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать заметку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_note_button">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>250</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить заметку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_note_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>280</y>
     <width>371</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить заметку</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tags_list">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>351</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_tag">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>600</y>
     <width>371</width>
     <height>28</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Введите тег ...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="find_notes_by_tag_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>670</y>
     <width>371</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Искать заметки по тегу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unpin_tag_from_note_button">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>640</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Открепить от заметки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_tag_to_note_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>640</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить к заметке</string>
   </property>
  </widget>
  <widget class="QLabel" name="tags_label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Список тегов </string>
   </property>
  </widget>
  <widget class="QListWidget" name="notes_list">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
